src/Microsoft.Data/DataFrame.BinaryOperations.cs

```diff
@@ -506,6 +552,147 @@ public DataFrame ElementwiseLessThan<T>(T value)
             }
             return retDataFrame;
         }
-        #endregion
+
+        /// <summary>
+        /// Performs a reversed element wise addition on each column
+        /// </summary>
+        public DataFrame RAdd<T>(T value, bool inPlace = false)
```

Comment:
`RAdd` doesn't follow our naming guidelines. See [https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions).

Reply:
Hmmm, I thought `RAdd` followed our guidelines. Are you looking at the "favor readability over brevity" one? I could live with `ReverseAdd` I guess. This is my comment from before:

> `RAdd` follows our naming guidelines, but `ROr` sounds a little weird. `ReverseAdd/ReverseOr` sound better?

Reply:
✓ DO favor readability over brevity.

> The property name CanScrollHorizontally is better than ScrollableX (an obscure reference to the X-axis).

What is `R` in these methods? Just like the "obscure reference to the X-axis" above, it is hard to guess what is meant by a single letter.

And yes:

X DO NOT use abbreviations or contractions as part of identifier names.

Also applies here.

Reply:
I would spell out `Reverse` and keep it before the operation name.

`ReverseAdd`, `ReverseSubtract`, `ReverseDivide`, etc.